README.md

```diff
@@ -41,11 +41,26 @@ spec:
 ```
 
 Note that this is a normal configuration file except that the `namespace` is given as `{{ namespace }}`. This will be
-replaced by the actual namespace when the [setup script](#setup-script) is run.
+replaced by the actual namespace when the [setup script](#setup-script.py) is run.
 
 # Setup Script
 
-Details to come.
+Set up your MariaDB password using Kubernetes secrets.
```

Comment:
For ease of use, eventually the setup script should probably (1) run this command for the user and (2) auto-generate a nice secure password. This is fine for now but I think I'll file a bug to automate this step.

Reply:
Thanks, @oliverdain!